include/translated-object.php

```diff
@@ -188,13 +188,16 @@ protected function should_update_translation_group( $id, $translations ) {
 	 * @since 0.5
 	 *
 	 * @param int   $id           Object id ( typically a post_id or term_id ).
-	 * @param int[] $translations An associative array of translations with language code as key and translation id as value.
-	 * @return void
+	 * @param array $translations An associative array of translations with language code as key and translation id as value.
```

Comment:
Why did you change this doc line?

Reply:
I did a mistake while resolving a merge conflict.